src/services/job/SyncDataToHubSpotJob.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace acclaro\translations\services\job;
+
+use acclaro\translations\Constants;
+use Craft;
+use craft\queue\BaseJob;
+use GuzzleHttp\Client;
+
+class SyncDataToHubSpotJob extends BaseJob
+{
+    public function execute($queue): void
+    {
+        $currentUser = Craft::$app->getUser()->getIdentity();
+
+        if ($currentUser) {
+            $email = $currentUser->email;
+            $username = $currentUser->username;
+
+            $siteName = Craft::$app->sites->getPrimarySite()->getName();
+            $baseUrl = Craft::$app->sites->getPrimarySite()->getBaseUrl();
+
+            $data = [
+                'email' => $email,
+                'username' => $username,
+                'siteName' => $siteName,
+                'baseUrl' => $baseUrl
+            ];
+
+            $apiEndpointN8N = Constants::API_ENDPOINT_N8N;
+        
+            // Initialize the Guzzle HTTP client
+            $client = new Client();
+        
+            try {
+                // Make the POST request
+                $response = $client->post($apiEndpointN8N, [
+                    'headers' => [
+                        'Content-Type' => 'application/json',
+                    ],
+                    'json' => $data,
+                ]);
+        
+                // Check if the request was successful
+                if ($response->getStatusCode() === 200) {
+                    $responseData = json_decode($response->getBody()->getContents(), true);
+                    Craft::info('API call was successful: ' . json_encode($responseData), __METHOD__);
```

Comment:
We do not log this way, please refer other places on code on how we log specifically from plugin.

Reply:
I will just go ahead and remove the same - will not be needed - had added to test on local anyways.